car.go

```diff
@@ -79,13 +83,22 @@ func ReadHeader(br *bufio.Reader) (*CarHeader, error) {
 	return &ch, nil
 }
 
-func (cw *carWriter) WriteHeader(h *CarHeader) error {
+func WriteHeader(h *CarHeader, w io.Writer) error {
 	hb, err := cbor.DumpObject(h)
 	if err != nil {
 		return err
 	}
 
-	return util.LdWrite(cw.w, hb)
+	return util.LdWrite(w, hb)
+}
+
+func SizeHeader(h *CarHeader) (uint64, error) {
```

Comment:
maybe `HeaderSize()`? I know that `SizeHeader` fits in with the other `*Header()` names but "_something_ size" seems like it might make for an easier-to-grok API? otherwise this could be read as something like "a header containing this size"

Reply:
sure